src/Polymake/polymake_to_oscar.jl

```diff
@@ -4,13 +4,14 @@ function convert(p::Polymake.Polynomial{Polymake.Rational, Polymake.to_cxx_type(
                           parent::Union{MPolyRing, Nothing} = nothing)
     coeff_vec = convert(Vector{fmpq}, Polymake.coefficients_as_vector(p))
     monomials = Matrix{Int}(Polymake.monomials_as_matrix(p))
-    n_vars = length(monomials[:, 1])
-    # not sure if the numbering is the best choice but it matches Polymake
+    n_vars = ncols(monomials)
+
+    # variable labeling from polymake is preserved
     if isnothing(parent)
         parent, _ = PolynomialRing(QQ, "x" => 0:n_vars - 1, cached=false)
     end
 
-    return parent(coeff_vec, [monomials[:, i] for i in 1:ncols(monomials)])
+    return parent(coeff_vec, [monomials[i, :] for i in 1:nrows(monomials)])
```

Comment:
Just so I understand: this fixes a bug in this conversion function? Do we have a test case that verifies this fix has the desired effect (e.g. perhaps without it the coordinates in the new Plücker tests would be wrong)?

Reply:
i was going to add a test now that should should cover this

Reply:
This line has code coverage, so I assume your test for "Grassman Plücker Relations" covers it and this remark is resolved. :-)